tests/e2e_regression/ensure_consistent_config.sh

```diff
@@ -19,7 +19,7 @@ shopt -s expand_aliases
 alias yaml2json="python3 -c 'import sys,yaml,json; print(json.dumps(yaml.safe_load(str(sys.stdin.read()))))'" 
 
 # Compare
-for range in $(find "$directory" -type d); do
+for range in $(find "tests/e2e_regression" -maxdepth 1 -mindepth 1 -type d); do
```

Comment:
`./tests/e2e_regression/ensure_consistent_config.sh: line 26: directory: unbound variable`

Reply:
woa how did this used to work

Reply:
:raised_eyebrow: yeah the old code makes no sense

Reply:
changed in #665, this script now takes the name of a suite as a parameter instead of searching around